echo.go

```diff
@@ -460,7 +460,9 @@ func (e *Echo) File(path, file string) *Route {
 	})
 }
 
-func (e *Echo) add(method, path string, handler HandlerFunc, middleware ...MiddlewareFunc) *Route {
+// Add registers a new route for all HTTP methods and path with matching handler
```

Comment:
Sorry, my head's not int the game today :disappointed: 

Should it be "for an HTTP method..." or "for a HTTP method..."?